src/bcrhp/bcrhp/pkg/map_layers/mapbox_spec_json/overlays/indian-reserves-band-names/indian-reserves-band-names.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "Indian Reserves and Band Names",
```

Comment:
Please update with name requested in the ticket.

Reply:
I will update, I missed the edit that was made on the ticket for this specific detail.